geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/sortedset/AbstractZRevRangeByScoreIntegrationTest.java

```diff
@@ -216,11 +216,22 @@ public void shouldReturnRange_givenLeftParenOnlyForMinOrMax() {
   public void shouldReturnRange_boundedByLimit() {
     createZSetRangeTestMap();
 
-    assertThat(jedis.zrevrangeByScore(KEY, "10", "0", 0, 2))
+    /*
+     * Fails in Jedis 4.1.1 for both Geode and Native Redis, so must be a Jedis bug
+     * TODO submit GitHub issue for this
```

Comment:
You should probably go ahead and file this bug and remove this TODO

Reply:
It's not entirely clear to me that the localSocketAddress is actually necessary here. I think this may have been part of the thread name for debugging purposes or for trivial reasons. So I will delete the commented out code, but only file a bug if someone thinks it is necessary. The test works fine without it.

Reply:
This TODO comment was actually some leftover cruft and just needed to be deleted.